packages/frontend-asset-loader/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@govuk-one-login/frontend-asset-loader",
+  "version": "0.0.1",
+  "description": "Loads cached assets",
+  "main": "build/cjs/index.cjs",
+  "module": "build/esm/index.js",
```

Comment:
Oh can we get the exports block in here 🙏 
I think given we had to add it for the others we probably should do this for all packages going forwards. Example PR where Joe added them here: https://github.com/govuk-one-login/govuk-one-login-frontend/commit/517ec2130fbf13bb42c786c0bfd88efa3b2a607c